### ISSUE TITLE
Builtin server list is out of date

### ISSUE THREAD

**Issue by rossengeorgiev** [SteamRE/SteamKit#247]
Nearly half the servers on the list are not responding

```
nc: connect to 208.64.200.201 port 27017 (tcp) failed: Connection timed out
nc: connect to 208.64.200.201 port 27018 (tcp) failed: Connection timed out
nc: connect to 208.64.200.201 port 27019 (tcp) failed: Connection timed out
nc: connect to 208.64.200.201 port 27020 (tcp) failed: Connection timed out
nc: connect to 208.64.200.202 port 27017 (tcp) failed: Connection timed out
nc: connect to 208.64.200.202 port 27018 (tcp) failed: Connection timed out
nc: connect to 208.64.200.202 port 27019 (tcp) failed: Connection timed out
nc: connect to 208.64.200.203 port 27017 (tcp) failed: Connection timed out
nc: connect to 208.64.200.203 port 27018 (tcp) failed: Connection timed out
nc: connect to 208.64.200.203 port 27019 (tcp) failed: Connection timed out
nc: connect to 208.64.200.204 port 27017 (tcp) failed: Connection timed out
nc: connect to 208.64.200.204 port 27018 (tcp) failed: Connection timed out
nc: connect to 208.64.200.204 port 27019 (tcp) failed: Connection timed out
nc: connect to 208.64.200.205 port 27017 (tcp) failed: Connection timed out
nc: connect to 208.64.200.205 port 27018 (tcp) failed: Connection timed out
nc: connect to 208.64.200.205 port 27019 (tcp) failed: Connection timed out
Connection to 208.78.164.9 27017 port [tcp/*] succeeded!
Connection to 208.78.164.9 27018 port [tcp/*] succeeded!
Connection to 208.78.164.9 27019 port [tcp/*] succeeded!
Connection to 208.78.164.10 27017 port [tcp/*] succeeded!
Connection to 208.78.164.10 27018 port [tcp/*] succeeded!
Connection to 208.78.164.10 27019 port [tcp/*] succeeded!
Connection to 208.78.164.11 27017 port [tcp/*] succeeded!
Connection to 208.78.164.11 27018 port [tcp/*] succeeded!
Connection to 208.78.164.11 27019 port [tcp/*] succeeded!
Connection to 208.78.164.12 27017 port [tcp/*] succeeded!
Connection to 208.78.164.12 27018 port [tcp/*] succeeded!
Connection to 208.78.164.12 27019 port [tcp/*] succeeded!
Connection to 208.78.164.13 27017 port [tcp/*] succeeded!
Connection to 208.78.164.13 27018 port [tcp/*] succeeded!
Connection to 208.78.164.13 27019 port [tcp/*] succeeded!
Connection to 208.78.164.14 27017 port [tcp/*] succeeded!
Connection to 208.78.164.14 27018 port [tcp/*] succeeded!
Connection to 208.78.164.14 27019 port [tcp/*] succeeded!
```

[steamstat.us](https://steamstat.us/) is currently showing 100% for Steam CMs

**Comment by doge333** [comment]
#229 and #236 discuss this problem.

It looks like that calling `SteamDirectory.Initialize().Wait();` on program startup solves the issue.

**Comment by rossengeorgiev** [comment]
Absolutely. It's worth considering updating the builtin list for next release.

**Comment by psychonic** [comment]
Fixed by #293
